### PR TITLE
More matchers and web methods

### PR DESCRIPTION
**Pull request by alextwilson**

### CODE REVIEW

File: test/assert.js [hunk 1]
```diff
@@ -16,30 +15,79 @@ var it = function(description, testFunction) {
...
+			return("Fail: " + assertionToCheck + " is empty");
+		}
+  },
 
+	isEmptyArray: function(assertionToCheck) {
+    if (typeof assertionToCheck === "object" && assertionToCheck.length === 0) {
+      return ("Pass: is an empty array.");
+    } else if (typeof assertionToCheck != "object") {
+    	return ("Fail: is not an array");
+    } else if (typeof assertionToCheck === "object" && assertionToCheck.length != 0) {
```
Comment (SimonBao) [inline test/assert.js]: assertionToCheck.constructor.name === 'Array' ?

**Comment by reenz** [comment]
:shipit:

**Comment by SimonBao** [comment]
:shipit:

**Comment by Gleoman** [comment]
:shipit:

**Pull request merged**